testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveTransactionalTable.java

```diff
@@ -1113,9 +1113,9 @@ public void testOrcColumnTypeChange(boolean transactional)
             log.info("This shows that Hive see the old data after a column is widened");
             assertThat(onHive().executeQuery("SELECT * FROM " + tableName))
                     .containsOnly(row(111, "Katy", 57, "CA"), row(222, "Joe", 72, "WA"));
-            log.info("This shows that Trino gets an exception trying to widen the type");
-            assertQueryFailure(() -> onTrino().executeQuery("SELECT * FROM " + tableName))
-                    .hasMessageMatching(".*Malformed ORC file. Cannot read SQL type 'integer' from ORC stream '.*.age' of type BYTE with attributes.*");
+            log.info("This shows that Trino see the old data after a column is widened");
```

Comment:
Do we need the log ? Seems like it should be a code comment

Reply:
I there are similar logs in that test class. It could be handled as a follow-up PR